webviews/components/Card.svelte

```diff
@@ -20,6 +21,12 @@
       </Modal>
     </div>
   {:else if card.content && card.content.title}
-    <p>{card.content.title}</p>
+    <div style="display: flex; flex-direction: row; width: 100%; justify-content: space-between;">
+      <p>{card.content.title}</p>
+      <span style= "border-style: solid; border-radius: 5px; padding:0 5px;">issue</span>
```

Comment:
This issue span's height is the full body height, we should probably change this.

![image](https://user-images.githubusercontent.com/43996118/109308757-76344100-7868-11eb-9bca-02a6937d6199.png)

Reply:
One more point, if you hardcode "issue" it will be a problem: https://github.com/Open-Sourced-Olaf/vscode-github-projects/projects/1#card-55768717


Suggestion:
```suggestion
      <span style= "border-style: solid; border-radius: 5px; padding:0 5px;">card.content.__typename</span>
```

Reply:
Solved by a future commit.

Reply:
Yeah..I will take a look at it.